sdk/release/RELEASE.md

```diff
@@ -162,27 +154,93 @@ windows` inside the `daml-language-ad-hoc` project. This command prints IP
 
-1. Add `+` at the end of line 26 after `(PartyIdHint "Alice")` and verify that
-   you get an error on line 27.
+1. Add `+` at the end of line 11 after `allocateParty "Alice"`, verify that
+   you get an error on line 12, then remove the `+`.
+
+1. Right click `IOU` in `... createCmd Main.IOU with ...` on line 12 and click Go to definition.
+  1. Verify you are taken to main/daml/Main.daml (and not somewhere in a `.daml` directory)
+
+1. Close the `Main.daml` tab, open `multi-package.yaml`, remove `- ./main`, save the file, then go to definition on IOU again as per previous instruction.
+  1. Verify that now (after a small delay), you are taken to a copy of `Main.daml` in a `.daml` directory at the root of the workspace.
+
+1. Close VSCode and delete the `myproject` directory ready for DPM testing using:
+  `cd .. && rmdir /S /Q myproject`
+
+1. On the [assembly] PR, add the comment:
+
+   > Manual tests passed on Windows using Daml Assistant.
+
+# Windows (DPM)
+
+You will now also need to test the release using DPM, please follow the instructions below (they look similar, but have small tweaks, don't reuse commands from Daml).
+ONLY RUN THESE STEPS FOR 3.4+ VERSIONS RELEASED AFTER 2025/09/22
+
+1. DPM snapshots are usually published under a different name, and via a different mechanism. For weekly testing, you may not know what this different version string is.
+   Use the following to install the DPM sdk on windows.
+   If you do not know the version string, and are testing the weekly release, go to this URL for the latest SDK:
+   ```
+   https://get.digitalasset-staging.com/install/latest-windows.html
+   ```
+   If you do know the version string, interpolate it into this string as `${version}` and go to the resulting URL:
+   ```
+   https://get.digitalasset.com/install/windows/dpm-${version}-windows-amd64.exe
+
+   ```
+
+1. Be sure to use a new terminal (created after installation) for the following steps.
+   If running `dpm` fails with a "cannot find command" error, run `setx /M PATH "%PATH%;%APPDATA%\dpm\bin"` in an administrator command prompt, and restart your existing terminals.
+
+1. Run `dpm versions` and verify the (green) version is what you expect. If you did not know what version to expect, ensure it is less than a week old for weekly testing.
+   (If it is too old, leave a message about this in #project-dpm and pause testing until you get a response. #project-dpm is US based so consider this for timezones.)
+
+1. On the [assembly] PR, add the comment (replacing $VERSION with the active DPM version string from above):
+
+   > Testing using DPM Version: $VERSION
+
+1. Run `dpm new myproject --template multi-package-example` to create a new project and switch to it
+   `cd myproject`.
+
+1. Build all packages by running `dpm build --all`.
+
+1. Run `dpm sandbox --dar ./main/.daml/dist/myproject-main-1.0.0.dar`. Wait for "Canton sandbox is ready." (it may take a little while)
```

Comment:
Can we update "a little while" to something more specific like "up to 2 minutes, but usually 10 seconds"?